torch/onnx/_internal/exporter/_building.py

```diff
@@ -334,6 +336,11 @@ def _process_python_constants(
 
         dtype = _determine_input_dtype(param, arg, type_binding)
 
+        # float representation of complex numbers
+        if isinstance(arg, complex):
+            # Convert the complex number to a float
+            arg = [arg.real, arg.imag]
```

Comment:
Should we handle this in _get_or_create_constant? Just thinking

Reply:
Done